codeforamerica/vita-min
# FYST-1159 Add MD specific 1099G builder

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1159
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- MD has their own 1099G schema, despite being very similar
- I also reordered some `supported_docs` tests in `md_return_xml_spec` since I had gotten confused
## How to test?
- Use a persona that has `UnemploymentCompAmt` (laney or zeus)
- Create one or more 1099Gs
- See that there are expected `MD1099G` elements in the XML output
## Screenshots (for visual changes)
<img width="534" alt="image" src="https://github.com/user-attachments/assets/e68537bd-e7b0-4f65-a9a2-4daa86bdde3f">

Note: Laney seems to have a different xml error right now related to `AddressLine2Txt` being added even when blank
